auth0/resource_auth0_custom_domain.go

```diff
@@ -6,6 +6,7 @@ import (
 
 	"gopkg.in/auth0.v2"
 	"gopkg.in/auth0.v2/management"
+	"net/http"
```

Comment:
Could you move this import statement on top? It should be done as the `goimports` tool with stdlib first, then third party and finally first party. For example:

```
import (
	"net/http"

	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"

	"gopkg.in/auth0.v2"
	"gopkg.in/auth0.v2/management"
)
```